bimmer_connected/cli.py

```diff
@@ -117,7 +132,40 @@ def image(args) -> None:
 def send_poi(args) -> None:
     account = ConnectedDriveAccount(args.username, args.password, get_region_from_name(args.region))
     vehicle = account.get_vehicle(args.vin)
-    poi = PointOfInterest(args.latitude, args.longitude, name=args.name)
+    poi = PointOfInterest(args.latitude, args.longitude, name=args.name,
+                          street=args.street, city=args.city, postalCode=args.postalcode, country=args.country)
+    vehicle.send_poi(poi)
+
+
+def send_poi_from_address(args) -> None:
+    account = ConnectedDriveAccount(args.username, args.password, get_region_from_name(args.region))
+    vehicle = account.get_vehicle(args.vin)
+    address = [(str(' '.join(args.address)))]
+    try:
+        g = requests.get("https://nominatim.openstreetmap.org", params={"q": address, "format": "json","addressdetails": 1,"limit": 1},).json()[0]
+    except IndexError:
+        print('\nAddress not found')
+        sys.exit(1)
```

Comment:
Just realized, you should be able to call `exit(1)` without explicilty calling it from `sys`, therefore also be able to remove the import up top.